packages/visx-grid/src/grids/GridPolar.tsx

```diff
@@ -0,0 +1,179 @@
+import GridRadial from './GridRadial';
+
+import { CommonGridProps, GridScale } from '../types';
+
+export type GridPolarProps<Scale extends GridScale> = CommonGridProps & {
+  /**
+   * If specified, the arc of each radial grid line will have this thickness, useful for fills.
+   */
+  arcThickness?: number;
+  /**
+   * The class name applied to the angle grid group.
+   */
+  classNameAngle?: string;
+  /**
+   * The class name applied to all radial lines.
+   */
+  classNameRadial?: string;
+  /**
+   * The end angle of the arc of radial grid lines.
+   */
+  endAngle?: number;
+  /**
+   * The color applied to the fill of the radial lines.
+   */
+  fillRadial?: string;
+  /**
+   * Radius which determines the start position of polar lines
+   */
+  innerRadius?: number;
+  /**
+   * Classname applied to angle line path.
+   */
+  lineClassNameAngle?: string;
+  /**
+   * Classname applied to radial line path.
+   */
+  lineClassNameRadial?: string;
+  /**
+   * Style object set as the angle line path style attribute.
+   */
+  lineStyleAngle?: object;
+  /**
+   * Style object set as the radius line path style attribute.
+   */
+  lineStyleRadial?: object;
+  /**
+   * The number of angle ticks wanted for the grid. Note this is approximate due to d3's algorithm,
+   * you can use tickValues for greater control
+   */
+  numTicksAngle?: number;
+  /**
+   * The number of radial ticks wanted for the grid. Note this is approximate due to d3's algorithm,
+   * you can use tickValues for greater control
+   */
+  numTicksRadial?: number;
+  /**
+   * Radius which determines the end position of polar lines
+   */
+  outerRadius?: number;
+  /**
+   * A [d3](https://github.com/d3/d3-scale) or [vx](https://github.com/hshoff/visx/tree/master/packages/visx-scale)
+   * scale function used to generate the angle of polar lines.
+   */
+  scaleAngle: Scale;
+  /**
+   * A [d3](https://github.com/d3/d3-scale) or [vx](https://github.com/hshoff/vx/tree/master/packages/vx-scale)
+   * scale function used to generate the radius of radial lines.
+   */
+  scaleRadial: Scale;
+  /**
+   * The start angle of the arc of radial grid lines.
+   */
+  startAngle?: number;
+  /**
+   * The color applied to the stroke of the angle lines.
+   */
+  strokeAngle?: string;
+  /**
+   * The color applied to the stroke of the radial lines.
+   */
+  strokeRadial?: string;
+  /**
+   * The pattern of dashes for angle line stroke.
+   */
+  strokeDasharrayAngle?: string;
+  /**
+   * The pattern of dashes for angle radial stroke.
+   */
+  strokeDasharrayRadial?: string;
+  /**
+   * The pixel value for the width of the angle lines.
+   */
+  strokeWidthAngle?: string | number;
+  /**
+   * The pixel value for the width of the radial lines.
+   */
+  strokeWidthRadial?: string | number;
+  /**
+   * An array of values that determine the number and values of the angle ticks. Falls
+   * back to `scale.ticks()` or `.domain()`.
+   */
+  tickValuesAngle?: Array<number>;
```

Comment:
This won't necessarily be number (`ScaleInput` is available from `@visx/scale`)

```suggestion
  tickValuesAngle?: ScaleInput<AngleScale>[];
```